sdk/tests/conformance/misc/object-deletion-behaviour.html

```diff
@@ -376,6 +376,48 @@
 shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.bindBuffer(gl.ELEMENT_ARRAY_BUFFER, bufferElement)");
 shouldBeNull("gl.getParameter(gl.ELEMENT_ARRAY_BUFFER_BINDING)");
 
+if (contextVersion > 1) {
```

Comment:
Could you please create a new conformance2/misc/object-deletion-behaviour.html and put the tests of these new object types in there? In general we have adhered to the policy of not putting WebGL 2.0-specific tests into conformance/, but rather into conformance2/. Thanks.